docs/_includes/layouts/page.html

```diff
@@ -9,7 +9,7 @@
     <div class="d-flex mx-auto w100 wmx12">
         {% include 'navigation.html' %}
 
-        <main id="content" class="d-flex fl-grow1 ps-relative pt96 pb24 pl48 md:pl24 sm:pl16 sm:pr16">
+        <main id="content" class="d-flex fl-grow1 ps-relative t64 py24 pl48 md:pl24 sm:pl16 sm:pr16">
```

Comment:
@dancormier I will reintroduce this as it was changed. Please refer to this comment for more details: https://github.com/StackExchange/Stacks/pull/1692#discussion_r1561284307